src/containers/main/Airdrop/AirdropGiftTracker/sections/CommunityMessages/CommunityMessages.tsx

```
@@ -0,0 +1,61 @@
+                    animate={{
+                        scale: 1,
+                        opacity: 1,
+                    }}
+                    exit={{
+                        scale: 0.5,
+                        opacity: 0,
+                    }}
+                    key={activeMessage.id}
+                    $type={activeMessage.type}
+                >
+                    <div dangerouslySetInnerHTML={{ __html: activeMessage.textHtml }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)